config/config.go

```diff
@@ -76,6 +76,9 @@ type Config struct {
 	// Monitoring defines information for metrics
 	Monitoring *metrics.MonitoringConfig `json:"monitoring,omitempty"`
 
+	// LoggerConfig configures the simple logger
+	LoggerConfig simple.Config `json:"logger,omitempty"`
```

Comment:
`simple.Config` -> `*simple.Config`

Reply:
Making this a value instead of pointer was an intentional choice.

If made a pointer, then the `nil` case has to be handled on logger instantiation.

```go
if config == nil {
  config = &Config{}
}
```

Reply:
you can do something like this here if you dont want to deal with nils
https://github.com/teslamotors/fleet-telemetry/blob/main/config/config_initializer.go#L44

```go
config := &Config{LoggerConfig: &simple.Config{}}
```

would prefer keeping pointers everywhere